lambda/main_test.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import "testing"
+
+func Test_toIntWithDefault(t *testing.T) {
+	type args struct {
+		value    string
+		defaults int
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    int
+		wantErr bool
```

Comment:
I generally advise to split test tables into error and non-error tables, in order to avoid complications like `(err != nil) != tt.wantErr`. This one is borderline (there's only 3 cases in total), but maybe that's an argument against a table-driven test in this case (or even unit testing `toIntWithDefault` at all)?

Reply:
Yeah I think what i have no used is a bit clearer and follows the original github.com/cweill/gotests/gotests template.

Reply:
I know the test is a bit much but I wanted to double check it so 🤷🏻